tensorflow/contrib/ffmpeg/kernels/ffmpeg_lib_test.cc

```diff
@@ -0,0 +1,64 @@
+// Copyright 2016 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// =============================================================================
+
+#include "tensorflow/contrib/ffmpeg/kernels/ffmpeg_lib.h"
+
+#include <stdlib.h>
+
+#include "tensorflow/core/lib/core/command_line_flags.h"
```

Comment:
Hi @fredbertsch: we are trying hard to get rid of command_line_flags from libraries --- can you remove the need for this?